.github/workflows/docker.yaml

```diff
@@ -35,6 +35,12 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          username: ${{ github.actor }}
```

Comment:
Whoops, I missed this.  Do we need `registry: ghcr.io` as well?  Not sure if it tries to log into docker.io by default or not

Reply:
Yep, added!